pandas/tests/frame/test_constructors.py

```diff
@@ -1643,6 +1643,11 @@ def test_constructor_empty_with_string_dtype(self):
         df = DataFrame(index=[0, 1], columns=[0, 1], dtype="U5")
         tm.assert_frame_equal(df, expected)
 
+        # GH 34915
```

Comment:
Could you create a separate test function?

Reply:
Done